codersdk/deployment_test.go

```diff
@@ -565,3 +565,38 @@ func TestPremiumSuperSet(t *testing.T) {
 	require.NotContains(t, enterprise.Features(), "", "enterprise should not contain empty string")
 	require.NotContains(t, premium.Features(), "", "premium should not contain empty string")
 }
+
+func TestRemoveTrailingVersionInfo(t *testing.T) {
+	t.Parallel()
+
+	testCases := []struct {
+		Name                       string
```

Comment:
```suggestion
```

Reply:
I forgot to name the tests 🤦 zero values smh.